frontend/src/components/regions/UserActions/ActivityLog.jsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import Typography from "@material-ui/core/Typography";
+import TodayIcon from "@material-ui/icons/Today";
+import Grid from "@material-ui/core/Grid";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme) => ({
+  containerStyles: {
+    width: "50%",
+    margin: 5,
+    border: "1px black solid",
+  },
+
+  fillerStyles: {
+    height: "100%",
+    textAlign: "right",
+    display: "flex",
+    flexDirection: "row",
+  },
+  labelStyles: {
+    width: 20,
+    color: "black",
+  },
+  content: {
+    width: "90%",
+    ["@media (max-width:620px)"]: {
+      fontSize: "1rem",
+    },
+  },
+  calendarIcon: {
+    margin: theme.spacing(1),
+  },
+  dateTypography: {
+    marginTop: theme.spacing(2),
+    ["@media (max-width:620px)"]: {
+      fontSize: "1rem",
+    },
+  },
+}));
+
+export default function ActivityLog({ eventList, sortedTimestampArray }) {
+  const classes = useStyles();
+
+  return (
+    <div>
+      {sortedTimestampArray.map((timestamp) => (
+        <>
+          <Typography variant="h6" className={classes.dateTypography}>
+            <TodayIcon className={classes.calendarIcon} />
+            {new Date(timestamp).toDateString()}
+          </Typography>
+          {eventList.map((item) => (
+            <>
+              {item.timestamp === timestamp && (
+                <Grid>
+                  <Grid
+                    className={classes.containerStyles}
```

Comment:
The wireframe was very low fidelity and shouldn't have been taken so literally.

We do need to see colors. How about using something like a Chip to show the color AND name the event.  It would be useful to be able to read what happened.

Does it really make sense to use a grid for this? Would a Paper not work better? Or a Card?  Grids are for responsive layout, this is a misuse.